.github/ISSUE_TEMPLATE/1-lts-release-checklist.md

```diff
@@ -70,7 +70,7 @@ This role should rotate between LTS releases
 
 - [ ] Assure that contents of master branch have been merged to the stable branch in the [packaging repository](https://github.com/jenkinsci/packaging), e.g. `stable-2.361`
 
-- [ ] Announce the start of the LTS release process in the #jenkins-release and #jenkins-infra IRC channels
+- [ ] Announce the start of the LTS release process in the [#jenkins-infra IRC channel](https://matrix.to/#/#jenkins-infra:libera.chat).
```

Comment:
No need to announce anymore on #jenkins-release too?

Reply:
The announcement is for the infra team (only afaik), I find the infra channel is the most appropriate one then, no?

Reply:
Sure, that's a great recent addition we're appreciating, just wondering about the #jenkins-release annoucement as it was part of the original checklist, and this PR would remove it.

Reply:
Added it again 👍🏻